lib/datasource/packagist/index.spec.ts

```diff
@@ -3,7 +3,7 @@ import * as httpMock from '../../../test/http-mock';
 import { loadJsonFixture } from '../../../test/util';
 import * as _hostRules from '../../util/host-rules';
 import * as composerVersioning from '../../versioning/composer';
-import { id as versioning } from '../../versioning/loose';
+import { id as versioning } from '../../versioning/composer';
```

Comment:
Because I can't find any usage of `loose` for Composer manager

Reply:
It's already imported and line above, so please combine imports.